### PR TITLE
[docs] Add prose docs on multiple event inputs

### PR DESCRIPTION
**Pull request by econchick**
I noticed we didn't have any prose docs on multiple event inputs so I quickly drafted some up 😄 

## Checklist for PR author(s)

- [ ] Format the pull request title like `[cli] Fixes bugs in 'klio job fake-cmd'`.
- [ ] Changes are covered by unit tests (no major decrease in code coverage %) and/or integration tests.
- [ ] Document any relevant additions/changes in the appropriate spot in `docs/src`.

### CODE REVIEW

File: docs/src/userguide/io/index.rst [hunk 1]
```diff
@@ -88,8 +88,66 @@ object, and it will be the responsibility of your own code to setup the input.
...
+  job_config:
+    events:
+      inputs:
+        - type: file
+          location: ./batch_track_ids_1.txt
+        - type: file
+          location: ./batch_track_ids_2.txt
+    # <-- snip -->
+
+Klio will automatically read from the configured inputs.
```
Comment (shireen-bean) [inline docs/src/userguide/io/index.rst:118]: Does it read them in order or simultaneously?

Comment (shireen-bean) [reply docs/src/userguide/io/index.rst:118]: oh looks like this is covered below!

Comment (econchick) [reply docs/src/userguide/io/index.rst:118]: It is probably a good idea to clarify it up here even if it's covered below, just to be super clear for the reader. Good catch!

**Review by shireen-bean — APPROVED** [review]
This sounds clear to me!

Tangential to this - I wonder if it would make sense to have a section to organize the different run.py arguments depending on configuration somewhere. Ex: the first argument into `def run(x, y)` could be a single input p_coll, it could be multiple `pcolls` with multiple inputs, and in batch it could be a pipeline object correct? Those things are all documented in our docs in separate logical places but im wondering if it would make sense in in the overview or anotomy::run.py section too.

**Comment by econchick** [comment]
@shireen-bean I think that's a good idea. I'll take that into account for when I continue my docs re-organization :) (also, it could be a pipeline object in either batch or streaming; it's when the user turns off klio's automatic handling of event inputs)

**Pull request merged**